src/parser.go

```diff
@@ -0,0 +1,57 @@
+package tome
+
+import (
+	"os"
+	"strings"
+)
+
+var batchSizeKb int64 = 512 * 1000
+
+// Parser interface
```

Comment:
Why not move the commit where you add linting before this, then you don't need to change it later?